ocs_ci/ocs/cluster.py

```diff
@@ -266,8 +266,8 @@ def cluster_health_check(self, timeout=None):
         if (
```

Comment:
This has been taken care of in Filip's ROSA PR.

Reply:
Specifically here - https://github.com/red-hat-storage/ocs-ci/pull/5005/files#diff-a4d73d82c94d13634aded838ea2490c117996218f405a8da15f9ecea093f3f6dR267